### PR TITLE
Search box rendering issue in Firefox and IE

### PR DESCRIPTION
**Pull request by richardimaoka**
Refs #21779

* Moving the version text 
  * less elements in breadcrumb -> less chances for overlaps among elements
* flexbox instead of float
  * [caniuse](http://caniuse.com/#feat=flexbox) says IE 8 doesn't support flexbox
  * However, IE doesn't even support algoria search? If search doesn't work, do we care about IE?

//Chrome on Windows
image

//IE 11 on Windows
image

//FireFox on Windows
image

### CODE REVIEW

File: akka-docs/_sphinx/themes/akka/static/docs.css [hunk 1]
```diff
@@ -226,13 +226,6 @@ p.caption { width: 80%; text-align: justify; font-size: 0.95em; font-style: ital
   text-decoration: none;
 }
 
-.algolia-autocomplete {
-  display: block;
-  width: 100%;
-  height: 100%;
-  top: -5px;
-  left: 10px;
-}
```
Comment (richard-imaoka) [inline akka-docs/_sphinx/themes/akka/static/docs.css]: It seems IE doesn't render the algolia search box. So it is a plain &lt;input&gt; in IE. 

So moving top -5px to breadcrumb input. That consistently works across browsers.

**Review by johanandren — APPROVED** [review]
LGTM, thanks for investigating and fixing @richard-imaoka, wrt IE I think that is enough for now (we will anyways spend some time on a new design soon).

**Comment by patriknw** [comment]
LGTM, thanks for quick fix @richard-imaoka

**Comment by ktoso** [comment]
LGTM, thanks!

**Pull request merged**